docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -987,6 +987,18 @@ expensive when the number of yb-tservers, or the number of databases goes up.
 
 {{< /note >}}
 
+## Auto Analyze Service flags
+
+Auto Analyze service flags are {{<tags/feature/tp>}}.
```

Comment:
If this feature is still TP, @yifanguan, could you add an idea number to this feature tag, if there is one? See this example on how to do it: https://github.com/yugabyte/yugabyte-db/blob/3b2bac3e9288c3c68af9b89fb1c42fb8b87dbf0e/docs/content/stable/develop/best-practices-ysql.md?plain=1#L187C1-L187C358

Reply:
Could you do the same in other places if you have mentioned feature tags in changes?